pkgs/applications/video/kodi/addons/joystick/default.nix

```diff
@@ -2,13 +2,13 @@
 buildKodiBinaryAddon rec {
   pname = namespace;
   namespace = "peripheral.joystick";
```

Comment:
not sure how to install this one, tried this but complains about package not found:
```
(pkgs.kodi.passthru.withPackages (kodiPkgs: with kodiPkgs; [
      peripheral.joystick
    ]))
```